lib/water_drop/message.rb

```diff
@@ -6,6 +6,7 @@ class Message
     # @param topic [String, Symbol] a topic to which we want to send a message
     # @param message [Object] any object that can be serialized to a JSON string or
     #   that can be casted to a string
+    # @param options [Hash] (optional) additonal options to pass to the Kafka producer
```

Comment:
@mensfeld Sorry missed that, thanks for reminding me. Is this somewhat coherent with the rest of the style?